src/zennit/composites.py

```diff
@@ -178,60 +182,98 @@ def __init__(self, low, high, canonizers=None):
 
 
+@register_composite('epsilon_alpha2_beta1')
+class EpsilonAlpha2Beta1(EpsilonAlphaBeta):
+    '''An explicit composite using the alpha2-beta1 rule for all convolutional layers and the epsilon rule for all
+    fully connected layers.
+    epsilon: float
+        Epsilon parameter for epsilon rule.
+    '''
+    def __init__(self, epsilon=1e-6, canonizers=None):
+        super().__init__(alpha=2, beta=1, canonizers=canonizers)
+
+
 @register_composite('epsilon_plus_flat')
 class EpsilonPlusFlat(SpecialFirstLayerMapComposite):
     '''An explicit composite using the flat rule for any linear first layer, the zplus rule for all other convolutional
     layers and the epsilon rule for all other fully connected layers.
+    epsilon: float
+        Epsilon parameter for epsilon rule.
     '''
-    def __init__(self, canonizers=None):
+    def __init__(self, epsilon=1e-6, canonizers=None):
         layer_map = LAYER_MAP_BASE + [
             (Convolution, ZPlus()),
-            (torch.nn.Linear, Epsilon()),
+            (torch.nn.Linear, Epsilon(epsilon)),
         ]
         first_map = [
             (Linear, Flat())
         ]
         super().__init__(layer_map, first_map, canonizers=canonizers)
 
 
-@register_composite('epsilon_alpha2_beta1_flat')
-class EpsilonAlpha2Beta1Flat(SpecialFirstLayerMapComposite):
-    '''An explicit composite using the flat rule for any linear first layer, the alpha2-beta1 rule for all other
+@register_composite('epsilon_alpha_beta_flat')
+class EpsilonAlphaBetaFlat(SpecialFirstLayerMapComposite):
+    '''An explicit composite using the flat rule for any linear first layer, the alpha-beta rule for all other
     convolutional layers and the epsilon rule for all other fully connected layers.
+    epsilon: float
+        Epsilon parameter for epsilon rule.
+    alpha: float
+        Alpha parameter for alpha-beta rule.
+    beta: float
+        Beta parameter for alpha-beta rule.
     '''
-    def __init__(self, canonizers=None):
+    def __init__(self, epsilon=1e-6, alpha=2, beta=1, canonizers=None):
         layer_map = LAYER_MAP_BASE + [
-            (Convolution, AlphaBeta(alpha=2, beta=1)),
-            (torch.nn.Linear, Epsilon()),
+            (Convolution, AlphaBeta(alpha=alpha, beta=beta)),
+            (torch.nn.Linear, Epsilon(epsilon)),
         ]
         first_map = [
             (Linear, Flat())
         ]
         super().__init__(layer_map, first_map, canonizers=canonizers)
 
 
+@register_composite('epsilon_alpha2_beta1_flat')
+class EpsilonAlpha2Beta1Flat(EpsilonAlphaBetaFlat):
+    '''An explicit composite using the flat rule for any linear first layer, the alpha2-beta1 rule for all other
+    convolutional layers and the epsilon rule for all other fully connected layers.
+    epsilon: float
+        Epsilon parameter for epsilon rule.
+    '''
+    def __init__(self, epsilon=1e-6, canonizers=None):
+        super().__init__(alpha=2, beta=1, canonizers=canonizers)
+
+
```

Comment:
Could you restore the original rule without the alpha, beta parameters, but with the epsilon?

Also use [numpydocstyle](https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring) so Sphinx can correctly parse the docstring.